### ISSUE TITLE
Beta7 crashes with portable config file

### ISSUE THREAD

**Issue by teconmoon** [DerKoun/bsnes-hd#3]
With bsnes, you can create a portable emulator folder by moving the settings.bml file to the same folder as the bsnes exe. This keeps the emu from storing the config in %LOCALAPPDATA%, and if you move the folder to another machine or user account you don't lose your config. Starting with beta 7, if the settings.bml file exists alongside the exe, bsnes crashes when opening any of the settings menus (Video, audio, input, etc). This isn't the case when the settings.bml file is in its default directory of %LOCALAPPDATA%\bsnes\

This issue doesn't occur with bsnes-hd beta 6 or prior. Please let me know if I can provide more details. Thanks for reading!

**Comment by DerKoun** [comment]
Thanks for reporting.
Could you also try with empty bml files, like in the release zip?
If that doesn't crash, could you upload your bml file(s) that did?

**Comment by teconmoon** [comment]
Hi there, thanks for the reply. Trying to duplicate this issue on other machines and not able to do so currently (Which I should have tried before submitting a ticket 🤦‍♂).

Originally, I had tried testing using the blank settings.bml file, and as I closed and re-opened the emulator I had about a 50% chance of it working or crashing when opening the settings menu. Tried the portable folder in multiple locations to ensure it wasn't a specific MicroSD card or hard drive that was causing me grief.

But as of now, on the original machine I was having the issue with, I am also no longer able to duplicate the issue despite some pretty drastic configuration changes (I maintain two emulator folders, bsnes & bsnes-hd. I replace the paths with relative paths to point to patches, saves, etc stored in the OG bsnes folder, ie ../bsnes/patches/). So for now this can be closed since it can't be consistently duplicated, if I find something that does cause this to be consistently duplicatable I will re-open the issue.
